src/Data/Argonaut/Encode/Class.purs

```diff
@@ -48,9 +48,12 @@ instance encodeJsonJString :: EncodeJson String where
 instance encodeJsonJson :: EncodeJson Json where
   encodeJson = identity
 
-instance encodeJsonChar :: EncodeJson Char where
+instance encodeJsonCodePoint :: EncodeJson CodePoint where
   encodeJson = encodeJson <<< singleton
 
+instance encodeJsonChar :: EncodeJson Char where
+  encodeJson = encodeJson <<< codePointFromChar
```

Comment:
You can just use the CodeUnit singleton here.

Reply:
ok thanks - gonna change this